README.md

```diff
@@ -0,0 +1,50 @@
+# NevolutionX
+Original Xbox dashboard created with nxdk.
+
+![Screenshot](https://en.wikipedia.org/wiki/Hammer_and_sickle#/media/File:Hammer_and_sickle_red_on_transparent.svg "Screenshot")
```

Comment:
I'm not convinced that a screenshot is a good idea and if it was, now is not the time as NevolutionX is far from as pretty as I envision it. :)

Reply:
I thought you might answer that. 🤔 There should be a readme nonetheless, I will remove the screennshot with the next commit.